bazel/foreign_cc/BUILD

```diff
@@ -147,8 +147,8 @@ envoy_cmake_external(
     deps = [
```

Comment:
We don't upstream LLVM/WAVM support, so changes to this file can be ignored.

Reply:
Done.